pkg/system/phase2_creating.go

```diff
@@ -27,6 +27,13 @@ func (r *Reconciler) ReconcilePhaseCreating() error {
 		"noobaa operator started phase 2/4 - \"Creating\"",
 	)
 
+	// A failiure to discover OAuth endpoints should not fail the entire reconcile phase.
+	oAuthEndpoints, err := util.DiscoverOAuthEndpoints()
+	if err != nil {
+		r.Logger.Errorf("Discovery of OAuth endpoints failed, got:", err)
```

Comment:
This is the line of the CI failure - Errorf requires `%s` in the message for the error arg

Reply:
Not specifically the whole operation is optional. Many env will not support the .well-known endpoint: Vennila k8s, OpenShift prior to 4

Reply:
Ah that's right.
So can you make sure this http call doesn't hang when running on vanilla kube?